l2geth/core/genesis.go

```diff
@@ -510,21 +480,17 @@ func DeveloperGenesisBlock(period uint64, faucet, l1XDomainMessengerAddress comm
 		if err != nil {
 			panic(fmt.Sprintf("Cannot fetch state dump: %s", err))
 		}
-		_, ok := stateDump.Accounts["Lib_AddressManager"]
```

Comment:
There are some other files that must be in the state dump too:
1. `OVM_ETH`
2. `OVM_L2CrossDomainMessenger`
3. `OVM_L1MessageSender`
4. `OVM_L2ToL1MessagePasser`

Is this the right place to make those assertions?

Reply:
Yes, good call